libs/coin-modules/coin-tron/src/logic/utils.ts

```diff
@@ -241,8 +241,8 @@ export const formatTrongridTxResponse = (
       switch (type) {
         case "VoteWitnessContract":
           return {
-            votes: votes.map(v => ({
-              address: encode58Check(v.vote_address),
+            votes: votes.map((v: any) => ({
```

Comment:
I feel like we should instead change line 200 to `tx.raw_data_contract[0].parameter.value` instead of using `get` the types will be infered